torch/_dynamo/symbolic_convert.py

```diff
@@ -897,7 +900,7 @@ def SETUP_WITH(self, inst):
         self.push(ctx.enter(self))
 
     def SETUP_FINALLY(self, inst):
-        self.block_stack.append(BlockStackEntry(inst.target))
+        self.block_stack.append(BlockStackEntry(0, inst.target))
```

Comment:
For all these functions that don't exist for 3.11, should we comment them the same way the ones only existing for python<=3.7 are?